### ISSUE TITLE
Elo change in a draw

### ISSUE THREAD

**Issue by chlorofight** [RumbleMike/ValorantStreamOverlay#76]
I dont know if this is a issue with riote or with the program but i draw a game 14-14 and when u draw your rank shoudlnt change buttt on the tool it said i lost 11 elo

**Comment by BobbyESP** [comment]
u can lose ELO in draw. Maybe if u played bad, ur elo went down.

**Comment by chlorofight** [comment]
but it says raiting not changed in game so you cant lose elo or so the game says

**Comment by BobbyESP** [comment]
Yeah, but i think the game has like: from 0 to 5 points of change, the rank doesn't change. from 5 to 10, minor decrease, from 10 to 20 decrease and 20 to something, major decrease. I dont really know at all.

**Comment by chlorofight** [comment]
i lost 15 i think from the draw i think it works more like mmr so your mmr goes up or down so they say your rank doesnt change to make players feel better
